internal/tsc_wrapped/tsc_wrapped.ts

```diff
@@ -145,6 +146,25 @@ function runOneBuild(
     return false;
   }
 
+  const emitResults: tsickle.EmitResult[] =
```

Comment:
add a TODO for me to reconcile this with the google3 version.

eventually there should be only one tsc_wrapped.ts